tests/test_dependencies.py

```diff
@@ -54,7 +61,7 @@ def test_standalone_render_dependencies(self):
         rendered_raw = template.render(Context({}))
 
         # Placeholders
-        self.assertEqual(rendered_raw.count('<link name="CSS_PLACEHOLDER">'), 1)
+        self.assertEqual(rendered_raw.count('<link name="CSS_PLACEHOLDER"/>'), 1)
```

Comment:
Tiny change, but added `/` at the end of the placeholder for the dependencies CSS, so it's properly formatted.

Because otherwise, as I used beautiful soup to parse and serialize the HTML, BS4 added the trailing `/` there anyway upon re-serialization.

Reply:
Oh nice! I just tried it, and using `html5` will also make BS4 stop adding empty value (`=""`) to boolean attributes (e.g. `data-djc-id-a1bc40=""`).

But it also breaks around 30 tests where the `data-djc-id-a1bc40=""` will need to be updated to `data-djc-id-a1bc40`. So I would make this change in a separate PR for clarity.

Reply:
Happy to hear that using html5 as a formatter solves those small annoyances. Will also make things easier if we switch to another (possibly home-brewd) parser in the future.